### PR TITLE
Fixed the plugin.

### PR DESCRIPTION
**Pull request by TheDeibo**
I saw a problem, and I fixed the plugin. 

Problems fixed:

EVERYTHING

### CODE REVIEW

**Comment by TheDeibo** [comment]
I have fixed all the plugins issues. Before, i could't use it at all, but thanks to some editing, (and luck) I've managed to get it working. 

Also, changed the Join and Chat events from their own individual files to one file. It makes it easier to edit that way, and make it easier to work.

I may be slightly above noob, but I can still fix stuff xD

**Comment by jasonwynn10** [comment]
thanks!

File: src/LostTeam/events/Events.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        #I had to change it into telling the player directly, because I tried all other ways of making it public, but didn't work. It will work someday...
+        $player = $event->getPlayer();
+        $name = $player->getName();
+        $player->sendMessage(TF::RED . "LostBot" . TF::WHITE . " > Welcome " . $name . "!");
+    }
+    public function onChat(PlayerChatEvent $event){
+        $player = $event->getPlayer();
+        $name = $player->getName();
+        $msg = $event->getMessage();
+        if($msg === "Hellp"){
```
Comment (jasonwynn10) [inline src/LostTeam/events/Events.php]: typo? hellp isn't a word that I know of XD

Comment (TheDeibo) [inline src/LostTeam/events/Events.php]: Typo means anything from
- spelling mistakes
- grammar 
- punctuation

And to me, it even means putting brackets in the wrong place.

**Comment by jasonwynn10** [comment]
typos are a real pain... what happened to auto correct?

**Comment by TheDeibo** [comment]
I prefer NOT having auto-correct whilst coding. It would ruin the fun of debugging :smiley: :computer:

**Comment by jasonwynn10** [comment]
too true

**Pull request merged**